src/main/java/org/example/movieSpring/service/MovieService.java

```diff
@@ -0,0 +1,51 @@
+package org.example.movieSpring.service;
+
+import org.example.movieSpring.model.Movie;
+import org.example.movieSpring.repo.MovieRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class MovieService {
+
+    private final MovieRepo movieRepo;
+
+    @Autowired
```

Comment:
autowired should not be on the method but on the parameter, also on constructors the annotation can be omited

Reply:
done

Reply:
Could you explain why Autowired isnt needed in this case? I thought it was, what makes it so that its not needed

Reply:
in spring constructor based dependency injection is considered best practice. 
its a longer explanation so here is a link. https://docs.spring.io/spring-framework/reference/core/beans/dependencies/factory-collaborators.html
this is mainly about spring but there is a box discussing constructor-based vs setter-based. in general the spring team recommends constructor based di
and if you use constructor based and only have 1 constructor it can be left out. its a quality of life feature

Reply:
done